Felienne/hedy
# [FEATURE] Implements "Hedy's choice" to set favorites on /explore page

**Description**
In this PR we implement a new and exciting feature: Hedy's choice! Enabling the admin to set programs as favorite and show these at the top of the /explore page. When filtering the favorite programs are hidden to prevent overly complex design. The programs are marked with a `hedy_choice` boolean on the Programs table, filtered with an index key. The view is limited to the 4 most recent programs. The functionality is written to retrieve all, which might be useful later on to allow the admin to change the favorite programs on the admin page as well.

**Fixes**
This PR fixes #2521

**How to test**
Make sure you are an admin and navigate to the `/explore` page. Verify that the UI has changed and now each program container has a small "star button" enabling you to select/remove a program as favorite. The color should change dynamically when selecting accompanied by a modal.alert. For the actual "favorite box", reload the explore page.

**Screenshot**
![Schermafbeelding 2022-05-10 om 13 33 06](https://user-images.githubusercontent.com/38155424/167619969-ffe4c038-c6a9-4c29-b684-8ecb4898d3c9.png)
